hydromt_wflow/data/lulc/v1.0/corine_mapping.csv

```diff
@@ -0,0 +1,46 @@
+corine,description,landuse,vegetation_kext,land_manning_n,soil_compacted_fraction,vegetation_root_depth,vegetation_leaf_storage,vegetation_wood_storage,land_water_fraction,vegetation_crop_factor,vegetation_feddes_alpha_h1,vegetation_feddes_h1,vegetation_feddes_h2,vegetation_feddes_h3_high,vegetation_feddes_h3_low,vegetation_feddes_h4,erosion_usle_c
+111,Continuous urban fabric,111,0.6,0.015,0.9,256.4,0.1,0.03,0,1,1,0,-100,-400,-1000,-16000,0.001
+112,Discontinuous urban fabric,112,0.6,0.015,0.75,256.4,0.1,0.03,0,1,1,0,-100,-400,-1000,-16000,0.001
+121,Industrial or commercial units and public facilities,121,0,0.011,1,0,0,0,0,0.3,1,0,-100,-400,-1000,-16000,0
+122,Road and rail networks and associated land,122,0.6,0.015,0.5,106.8,0.1,0.03,0,1,1,0,-100,-400,-1000,-16000,0.001
+123,Port areas,123,0,0.011,1,0,0,0,0,0.3,1,0,-100,-400,-1000,-16000,0
+124,Airports,124,0.6,0.015,0.5,106.8,0.1,0.03,0,1,1,0,-100,-400,-1000,-16000,0
+131,Mineral extraction sites,131,0,0.011,0,0,0,0,0,0.3,1,0,-100,-400,-1000,-16000,0.35
+132,Dump sites,132,0,0.011,0.5,0,0,0,0,0.3,1,0,-100,-400,-1000,-16000,0.35
+133,Construction sites,133,0,0.011,0.5,0,0,0,0,0.3,1,0,-100,-400,-1000,-16000,0.35
+141,Green urban areas,141,0.6,0.15,0.1,256.4,0.15,0.06,0,1,1,0,-100,-400,-1000,-16000,0.001
+142,Sport and leisure facilities,142,0.6,0.15,0.5,106.8,0.1,0.03,0,1,1,0,-100,-400,-1000,-16000,0.001
```

Comment:
The compacted fraction for class 142 seems a bit high, given that Corine describes it as "Camping grounds, sports grounds, leisure parks, golf courses, racecourses, etc. Includes formal parks not surrounded by urban
areas.". I expect it to be closer/very similar to class 141

Reply:
You're right I'll update